docs/articles/documentation/extending-testcafe/reporter-plugin/helpers.md

```diff
@@ -224,7 +224,7 @@ Parameter   | Type   | Description
 The following example demonstrates how to escape the fixture name *Tests for the "Example" page* for HTML.
```

Comment:
how to escape? Why 'escape'?

Reply:
Because this function actually escapes strings (https://en.wiktionary.org/wiki/escape#Verb, meaning no 5). 

However, we can use 'encode' instead.

*The following example demonstrates how to encode the fixture name \*Tests for the "Example" page\* for HTML.*